runtime/src/iree-amd-aie/driver/xrt/direct_command_buffer.cc

```diff
@@ -360,7 +362,7 @@ static iree_status_t iree_hal_xrt_direct_command_buffer_dispatch(
     return iree_make_status(IREE_STATUS_UNKNOWN, e.what());
   }
 
-  for (xrt::bo& bo : bos) bo.sync(XCL_BO_SYNC_BO_FROM_DEVICE);
+  ofm_bo.sync(XCL_BO_SYNC_BO_FROM_DEVICE);
```

Comment:
Just because today we only have one output, doesn't mean tomorrow we will only have one output.

Reply:
So far all NPU products only use one output buffer.

Reply:
This has absolutely nothing to do with NPU and everything to do with the model.

Reply:
> This has absolutely nothing to do with NPU and everything to do with the model.

Do we have a model running on NPU with multiple output bo?

Reply:
This indeed doesn't have anything to do with NPU and whether we currently have a model like this or not. We shouldn't make the assumption that there is a single output buffer.

Reply:
NPU compute pipeline only support one output buffer.

Reply:
You're just completely wrong https://github.com/Xilinx/mlir-aie/blob/perf_plus_5x4/test/ipu-xrt/e2e/test_manual_dpu_args.py#L337

Reply:
Not sure the code snippet below is related to "NPU compute pipeline support one output buffer".
        for c, w in enumerate(wraps):
            if not np.array_equal(RANDOM_WEIGHT + c, w):
                with np.printoptions(threshold=sys.maxsize, linewidth=sys.maxsize):
                    print("RANDOM_WEIGHT", RANDOM_WEIGHT + c)
                    print(f"{buffer_args[c]} =", w)
                    assert False

What I see is that XRT driver currently only support one output buffer.

Reply:
Can we discuss this further offline?

Reply:
Currently we are patching-in-firmware, as a result, only one OFM is supported. Assume one day we will switch to patching-in-host, then we can use multiple OFM

Reply:
https://github.com/nod-ai/iree-amd-aie/blob/main/compiler/plugins/target/AMD-AIE/aie/AMDAIEDmaToNpu.cpp#L81